draft-bbs-signatures.md

```diff
@@ -748,6 +753,79 @@ Procedure:
 5. return (scalar_1, ..., scalar_n)
 ```
 
+### OctetsToSignature
+
+This operation describes how to decode an octet string, validate it and return the underlying components that make up the signature.
+
+```
+(A, e, s) = octets_to_signature(signature_octets)
+
+Inputs:
+
+- signature_octets (REQUIRED), octet string of the form output from signature_to_octets operation.
+
+Outputs:
+
+- A, a valid point in the G1 subgroup.
+- e, a non-negative integer representing a valid scalar value with the range of 0 < e < q.
+- s, a non-negative integer representing a valid scalar value with the range of 0 < e < q.
+
+Procedure:
+
+1. if len(signature_octets) != (octet_point_length + 2 * octet_scalar_length), return INVALID
+
+2. a_octets = signature_octets[0..(octet_point_length - 1)]
+
+3. A = octets_to_point(a_octets)
+
+4. if A is INVALID, return INVALID
```

Comment:
Whats the best syntax for checking whether A is infinity or not?

Reply:
We usually denote it as `A != 1_G1`

Reply:
So define `1_G1` as the point at infinity for the G1 subgroup?

Reply:
Using `Identity_G1` instead here as it already exists

Reply:
👍🏻